src/components/modals/help-modal.jsx

```diff
@@ -70,7 +70,7 @@ export default class HelpModal extends React.Component {
           <Tabs value={value} onChange={this.handleChange}>
             <Tab label="Keyboard shortcuts" />
             <Tab label="More resources" />
-            <Tab label="About Iodide" href="#basic-tabs" />
+            <Tab label="About Iodide" />
```

Comment:
the `href` here made this tab button render as an <a> which was not needed and caused clicking the tab label to open a new browser tab